modules/gapi/misc/python/package/gapi/__init__.py

```diff
@@ -297,3 +297,5 @@ def kernel_with_params(cls):
 cv.gapi.wip.draw.Poly = cv.gapi_wip_draw_Poly
 
 cv.gapi.streaming.queue_capacity = cv.gapi_streaming_queue_capacity
+
+cv.gapi.wip.gst.GStreamerPipeline = cv.gapi.wip.GStreamerPipeline = cv.gapi_wip_gst_GStreamerPipeline
```

Comment:
???

Reply:
Don't know but it's supported by ocv-version: https://github.com/opencv/opencv/blob/4.x/modules/gapi/include/opencv2/gapi/streaming/gstreamer/gstreamersource.hpp#L83

Reply:
I meant double assignments

Reply:
OK, removed `wip.gst.` assignment